cluster-provision/k8s/1.18/bind_device_to_vfio.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+set -ex
+
+if [ "$1" != "--vendor" ]; then
+    echo "No vendor provided"
+    exit 1
+fi
+vendor=$2
+
+function get_device_driver() {
+    local dev_driver=$(readlink $driver_path)
+    echo "${dev_driver##*/}"
+}
+
+# find the PCI address of the device by vendor_id:product_id
+pci_address=(`lspci -n -d ${vendor}`)
+pci_address="0000:${pci_address[0]}"
```

Comment:
See `lspci -D`.
Also, just the vendor is probably not enough to identify the device.

Reply:
Yeah, vendor:product is not enough in general, but in our case there will only be one device like that.

The trouble is that with q35 and in this pci topology we won't get the same pci address in the vm so we need to look it up in the vm by it's vendor:product ID.

Reply:
Oh my bad, vendor:product is actually (mostly) fine!
The name of the option confused me... Maybe pci-ids would be a better one?